.github/workflows/test-pr.yml

```diff
@@ -155,9 +155,8 @@ jobs:
         if: ${{ !startsWith(matrix.os, 'self') }}
         uses: cachix/cachix-action@v10
         with:
-          name: runtimeverification
-          extraPullNames: kore
-          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+          name: k-framework
+          authToken: '${{ secrets.CACHIX_PUBLIC_TOKEN }}'
 
```

Comment:
@jberthold we do set up the `extra_nix_config` here, but it doesn't run on our own CI machines, so I guess it's not actually using taht. We could add back `extraPullNames: kore`, but I'd like to consolidate caches.

Reply:
What I meant to suggest was to add the `https://cache.iog.io` substituter (IOHK Haskell infra cache), in `"Install Nix"`.
We do this about 8 lines above here but the `test-nix` job in  https://github.com/runtimeverification/k/blob/91f11338d4ccf3626f4e8874e10071e09d5131c6/.github/workflows/test-pr.yml#L192..L198 does not add that cache.
Thought it would be worth a try